wd_helper.au3

```diff
@@ -2213,6 +2213,7 @@ EndFunc   ;==>_WD_CheckContext
 ;                  in this function. You may refer to this cheatsheet for further information:
 ;                  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet
 ;                  The matching elements are returned in the same format as the results from _WD_FindElement
+;                  As JavaScript's RegEx engine differs from AutoIt's PCRE engine you can use https://regex101.com with FLAVOR set to: ECMAScript (JavaScript) to validate your RegEx
```

Comment:
This remark is should be either eliminated or combined with the existing remarks that already attempt to explain that Autoit RegEx <> Javascript RegEx

Reply:
I see no obstacles, so please propose the final version.

Reply:
See 1794062127aff5a41757431ef3ea1161d748c7be for my revisions